src/winforms/toga_winforms/widgets/box.py

```diff
@@ -21,3 +22,8 @@ def set_bounds(self, x, y, width, height):
             vertical_size_adjustment = self.interface.style.padding_bottom
             self.native.Size = Size(width + horizontal_size_adjustment, height + vertical_size_adjustment)
             self.native.Location = Point(x - horizontal_shift, y + vertical_shift)
+
+    def set_background_color(self, value):
+        if value is not None:
```

Comment:
If value *is* None, is there an obvious "revert to default color" option here? As I read this, if I set the color to `red`, and then set it to None, the box will remain red.